sdk/core/Microsoft.Azure.Core.NewtonsoftJson/README.md

```diff
@@ -0,0 +1,49 @@
+# Newtonsoft.Json implementation for Azure Core Experimental shared client library for .NET
```

Comment:
Let's start a thread, but my understanding is the name for this package should be `Microsoft.[Feature].[Dependency]` and would expect it to be more like `Microsoft.ObjectSerializer.NewtonsoftJson`.  We might also include the version if we want to avoid breaks.  +@KrzysztofCwalina

Reply:
I also think this should probably go in the `/sdk/extensions/` folder instead of `/sdk/core/`

Reply:
I disagree, we would need a place to put GeoJSON converter implementation for newtonsoft too. Not sure if it's wise to ship a package for every tiny feature. Or we can say the "feature" here is Azure.Core integration.

Reply:
I started a thread, but unless we reach agreement soon will keep this as-is for now.

Reply:
I think the GeoJSON converter belongs somewhere else.  We don't want everyone using Newtonsoft to have to pull in a reference to Microsoft.Spatial.  I'd prefer `Microsoft.ObjectSerializer.NewtonsoftJson` and `Microsoft.ObjectSerializer.MicrosoftSpatial` as separate packages.

Reply:
>We don't want everyone using Newtonsoft to have to pull in a reference to Microsoft.Spatial

I don't see why that would be the case.

Azure.Core.Newtonsoft would have 2 dependencies: Azure.Core and Newtonsoft.Json. And we would have an implementation of newtonsoft json convertor for out Azure.Core GeoJSON types.

>I'd prefer Microsoft.ObjectSerializer.NewtonsoftJson and Microsoft.ObjectSerializer.MicrosoftSpatial as separate packages.

I'm not sure I understand the `Microsoft.ObjectSerializer.MicrosoftSpatial` part and I'm afraid of package count explosion if we start shipping a package for every little feature that doesn't require an additional dependency.

Reply:
Ah, sorry, I thought you were talking about support for https://github.com/Azure/azure-sdk-for-net/issues/13165 as well.  We'll need System.Text.Json and Newtonsoft.Json converters for those spatial types.

I think `Microsoft.ObjectSerializer.NewtonsoftJson` could hold both an implementation of ObjectSerializer and Newtonsoft converters for any Azure.Core types.  Maybe `ObjectSerializer` isn't a broad enough feature name and we should use something more like `Microsoft.Serialization.NewtonsoftJson`?

Reply:
What if we do some JsonPatch support later that would require Newtonsoft adapters.

I would like the understand benefits of having feature-scoped packages instead of something more generic and  keeping a leeway for the future.

Reply:
I agree with @pakrym. I don't understand what benefit **not** having a 1:1 relationship with Azure.Core with an assembly for Newtonsoft.Json. Think of it this way: that 1 assembly is the Newtonsoft.Json implementation**s** for Azure.Core. If it weren't  already taboo, I'd actually think something like Azure.Core.Newtonsoft(.)Json would be appropriate, just for comparison.

Reply:
I would consider `ObjectSerializer` a feature and `Azure.Core` a library.  We have approval for `Microsoft.[Feature].[Dependency]`.  `Microsoft.Azure.Core.[Dependency]` feels confusing given all the `Microsoft.Azure.*` Track 1 libraries. 
 We should discuss it with the Krzysztof and the Arch Board.

Reply:
@AlexGhiondea also agrees with the current plane. @KrzysztofCwalina, what do you think?